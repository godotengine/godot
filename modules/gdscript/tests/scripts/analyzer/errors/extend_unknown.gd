class Foo:
	pass

class Bar extends Foo.Baz:
	pass

func test():
	print('not ok')
