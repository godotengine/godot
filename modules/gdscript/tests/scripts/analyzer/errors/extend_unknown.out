GDTEST_ANALYZER_ERROR
>> ERROR at line 4: Could not find nested type "Baz".
